roles/openshift_prometheus/vars/openshift-enterprise.yml

```diff
@@ -4,9 +4,11 @@ l_openshift_prometheus_image_prefix: "{{ openshift_prometheus_image_prefix | def
 l_openshift_prometheus_proxy_image_prefix: "{{ openshift_prometheus_proxy_image_prefix | default(l_openshift_prometheus_image_prefix) }}"
 l_openshift_prometheus_alertmanager_image_prefix: "{{ openshift_prometheus_altermanager_image_prefix | default(l_openshift_prometheus_image_prefix) }}"
 l_openshift_prometheus_alertbuffer_image_prefix: "{{ openshift_prometheus_alertbuffer_image_prefix | default(l_openshift_prometheus_image_prefix) }}"
+l_openshift_prometheus_node_exporter_image_prefix: "{{ openshift_prometheus_node_exporter_image_prefix | default(l_openshift_prometheus_image_prefix) }}"
 
 # image version defaults
 l_openshift_prometheus_image_version: "{{ openshift_prometheus_image_version | default('v3.7') }}"
 l_openshift_prometheus_proxy_image_version: "{{ openshift_prometheus_proxy_image_version | default('v3.7') }}"
 l_openshift_prometheus_alertmanager_image_version: "{{ openshift_prometheus_alertmanager_image_version | default('v3.7') }}"
 l_openshift_prometheus_alertbuffer_image_version: "{{ openshift_prometheus_alertbuffer_image_version | default('v3.7') }}"
+l_openshift_prometheus_node_exporter_image_version: "{{ openshift_prometheus_node_exporter_image_version | default('v0.14.0') }}"
```

Comment:
@pgier do you know what version we'll be releasing?

Reply:
It should be updated to v0.15.2